packages/framework/src/Component/Maintenance/MaintenanceModeSubscriber.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Maintenance;
+
+use Shopsys\FrameworkBundle\Component\Redis\RedisClientFacade;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Twig\Environment;
+
+class MaintenanceModeSubscriber implements EventSubscriberInterface
+{
+    public const MAINTENANCE_CACHE_KEY = 'maintenance';
```

Comment:
The word "cache" is a little confusing, it's not a cache.

Reply:
fixed